dallinger/experiment_server/experiment_server.py

```diff
@@ -811,15 +811,23 @@ def create_participant(worker_id, hit_id, assignment_id, mode, entry_information
 
     # Create the new participant.
     exp = Experiment(session)
-    participant = exp.create_participant(
-        worker_id,
-        hit_id,
-        assignment_id,
-        mode,
-        recruiter_name,
-        fingerprint_hash,
-        entry_information,
-    )
+    participant_vals = {
+        "worker_id": worker_id,
+        "hit_id": hit_id,
+        "assignment_id": assignment_id,
+        "mode": mode,
+        "recruiter_name": recruiter_name,
+        "fingerprint_hash": fingerprint_hash,
+        "entry_information": entry_information,
+    }
+    try:
+        participant = exp.create_participant(**participant_vals)
+    except Exception:
+        db.logger.exception(
+            "Error creating particant using these values: {}".format(participant_vals)
+        )
+        msg = "/participant POST: required values were missing or invalid."
```

Comment:
If we know this is the message, shouldn't we be catching some more specific exception? I guess what we return to the user is less important than what we log. This looks fine though.